src/main/java/Employee.java

```diff
@@ -0,0 +1,96 @@
+import javax.persistence.*;
+
+@Entity
+@Table(name = "employee")
+public class Employee {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
```

Comment:
```suggestion
    private Integer id;
```
Поменяй тип переменной на Integer